packages/app/src/compiler/index.ts

```diff
@@ -7,7 +7,7 @@ export async function compile(pathname: string, tsconfig: string): Promise<Pipel
   const project = new Project({ tsConfigFilePath: tsconfig });
   const script = project.getSourceFileOrThrow(pathname);
   // TODO: needs to verify if this imported the learnable.
-  const app = script.getImportDeclarationOrThrow('@pipcook/app');
+  const _app = script.getImportDeclarationOrThrow('@pipcook/app');
```

Comment:
We can remove the `app` declaration.

Reply:
Should the todo comment be removed together?

Reply:
Just left the TODO